plugins/module_utils/s3.py

```diff
@@ -151,3 +151,26 @@ def s3_extra_params(options, sigv4=False):
         config["signature_version"] = "s3v4"
     extra_params["config"] = config
     return extra_params
+
+
+# _list_backup_inventory_configurations is a workaround for a missing paginator for listing
+# bucket inventory configuration in boto3:
+# https://github.com/boto/botocore/blob/1.34.141/botocore/data/s3/2006-03-01/paginators-1.json
+def _list_bucket_inventory_configurations(client, bucket_name):
```

Comment:
While this is a work around, it's called from outside of module_utils and generally shouldn't be a "protected" method (ie. please drop the _ from the name)